libheif/heif_cxx.h

```diff
@@ -284,7 +284,7 @@ namespace heif {
 
     bool is_full_range() const;
 
-    void set_color_primaties(heif_color_primaries cp);
+    void set_color_primaries(heif_color_primaries cp);
```

Comment:
As noted, I could include the old name as a deprecated version.

C++ classes are brittle which makes things interesting. In some ways, it might make more sense to just force people to update when they upgrade. But, I'm not sure what the repository's policy is for C++...

Reply:
If you add the old name to the very end, it should not present any C++ ABI or API problem. In this case, it should work even if they're building with mismatched header/libs, since the signatures and output are identical. Just add a line with `// DEPRECATED, typo in function name` like the end of heif.h.

Reply:
Since there is no C++ shared library, there is no issue here. It will fail during compilation with old application source, but that is easy to fix. If there were a shared library, we would need ABI stability so that a newer installed library does not break existing binaries.